src/components/app/data/utils.js

```diff
@@ -272,12 +272,6 @@ export function getAssignmentsByState(assignments = []) {
  * @returns
  */
 export function transformEnterpriseCustomer(enterpriseCustomer) {
-  // If the learner portal is not enabled for the displayed enterprise customer, return null. This
-  // results in the enterprise learner portal not being accessible for the user, showing a 404 page.
-  // This logic needs to be maintained for individuals checks against the enterprise customer for staff users
-  if (!enterpriseCustomer.enableLearnerPortal) {
```

Comment:
checks against `!enableLearnerPortal` happen prior to calling this function. Now, this function is only called on enterprise customers with learner portal enabled.